code/processes/wdb.q

```diff
@@ -232,14 +232,26 @@ doreload:{[pt]
 	];
 	};
 
+// set .z.zd to control how data gets compressed
+setcompression:{[compression] if[ 3= count compression;
+							.lg.o[`compression;"setting compression level to (",(";" sv string compression),")"];
+                        				.z.zd:compression;
+                        				.lg.o[`compression;".z.zd has been set to (",(";" sv string .z.zd),")"]]}
+//reset .z.zd to 16 0 0 (i.e no compression)
+removecompression:{[compression] if[ 3= count compression;
+					    		.z.zd:16 0 0;
+                                            		.lg.o[`compression;"resetting compression level to (",(";" sv string 16 0 0),")"]]}
+
```

Comment:
Could you make "set compression" and removecompression a single function?  (or remove could be a projection of set).  So basically if count=3, then set it.  if count = 0 (empty list), then unset it